architecture/management_console.adoc

```diff
@@ -0,0 +1,21 @@
+= HTTP Remote Command Execution
```

Comment:
Thanks for these additions, @jwforres. I assume the title here is just leftovers from copying the topic template?